tailormap-components/projects/core/src/lib/user-interface/attributelist/attribute-list-panel/attribute-list-panel.component.ts

```diff
@@ -57,6 +58,8 @@ export class AttributeListPanelComponent implements OnInit, AfterViewInit, Layou
       .subscribe(tabs => {
         this.tabs = tabs;
       });
+    this.title$ = this.store$.select(selectAttributeListConfig)
```

Comment:
Why do this differently than the tooltip in attribute-list-button.component.ts? Title isn't going to change (I think?)?

I don't care which one is used, but I merely noticed the difference

Reply:
In the button, the config is passed through an @Input by Ext. Then its added to the state. Since we already have the config there I did not read it from the state as well.